nv* compilers show up as "PGI" but they don't like the p7-64 flag.
Instead, we use "host" always for the target architecture
author: Miguel Dias Costa
--- c_check.orig	2024-01-19 14:51:18.033217606 +0800
+++ c_check	2024-01-19 14:51:31.713311050 +0800
@@ -128,7 +128,7 @@
     PGI)
     	case "$BINARY" in
     	    32) compiler_name="$compiler_name -tp p7" ;;
-    	    64) compiler_name="$compiler_name -tp p7-64" ;;
+    	    64) compiler_name="$compiler_name -tp host" ;;
     	esac
     	openmp='-mp'
     	defined=1
--- c_check.pl.orig	2024-01-19 14:51:37.543350877 +0800
+++ c_check.pl	2024-01-19 14:51:53.513459955 +0800
@@ -158,7 +158,7 @@
 
 if ($compiler eq "PGI") {
     $compiler_name .= " -tp p7"    if ($binary eq "32");
-    $compiler_name .= " -tp p7-64" if ($binary eq "64");
+    $compiler_name .= " -tp host" if ($binary eq "64");
     $openmp = "-mp";
     $defined = 1;
 }
